doc/arch/adr-000-document-architectural-decisions.md

```diff
@@ -0,0 +1,21 @@
+# ADR 1: Document architectural decisions
```

Comment:
The file name is `adr-000` but you titled this `ADR 1`. Also the other ADRs have leading `00`s as in `ADR 002:`